### ISSUE TITLE
adding another print view in user view

### ISSUE THREAD

**Issue by zlyczarny** [grokability/snipe-it#9496]
- Snipe-IT 5.1.2

Now in the user view, there's only one print view (Print all assigned assets), in my company we always use two or more layouts of assigning assets, like list of assets assigned to new worker which he gets on the first day to sign. On the last day of work, he needs to sign another list of assets that need to be taken back. The third layout consists of access to systems for that user (GDPR purpose). Probably having a few layout would be also useful for others.

Best solution would be add 1 or 2 layouts in user view, it could even be the same as print.blade right now, but in different file so every layout could be changed.

I found out that print button is stored at view\view.blade.php (line 340) which has route to 'users.print'.  
It would be great if someone could help me with that.

**Comment by UserGoodtm** [comment]
I agree that a similar function is needed. For example, to be able to print out some instructions to a new user

**Comment by diegogarciadev** [comment]
+1

**Comment by cliff-LINKS** [comment]
+1
